README.md

```diff
@@ -16,6 +16,11 @@ meson build --prefix=/usr --buildtype=release
 ninja -C build && sudo ninja -C build install
 ```
 
+If you want to use a system-wide installation of `wf-config`, just add `-Duse_system_wfconfig=enabled` to the second line.
+
+If you want to build wlroots automatically or you want to use the one provided by Wayfire, add `-Duse_system_wlroots=disabled` to the second line. This will also overwrite any previous installations in the destination that you have. 
+
```

Comment:
I had to read these comments a few times to understand them, I think they could stand to be clarified a bit more. But the real problem I see is, as far as I understand it, that package maintainers can't really use -Duse_system_wlroots=disabled because then wayfire would conflict with the wlroots package because they would both provide the same files. And if package maintainers want to package wf-config, then if wayfire is built with -Duse_system_wfconfig=disabled, it would conflict with the wf-config package the same way. So either we should leave a note to maintainers saying that they should use -Duse_system_w*=enabled to package and 'disabled' is for users that are building manually or provide some solution to rectify the situation.

Reply:
Wait what? I agree the second line is confusing, I'll edit this, but I dont' understand what the problem with the rest is.

Current default behavior: Build wf-config as a submodule. If you are a packager, you "want to use system wf-config", so just use the option.

Also default: do not build the wlroots submodule. This is reasonable since I expect a lot of people already have wlroots installed/packaged.